src/platforms/apple/common/configuration/watchdog-terminations.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Watchdog Terminations
+sidebar_order: 10
+description: "Learn how to turn off Watchdog Termination tracking"
+---
+
+This integration tracks <a href="https://developer.apple.com/documentation/xcode/addressing-watchdog-terminations">watchdog terminations</a> based on heuristics. This feature is available for iOS, tvOS, and Mac Catalyst, works only if the application was in the foreground, and doesn't track watchdog terminations for unit tests.
+
+When a typical unhandled error occurs, the Apple SDK writes a report to disk with the current state of your application, with details like the stack trace, tags, breadcrumbs, and so on, before the app terminates. When the watchdog terminates your app this happens without further notice, which means the SDK can't write a report to disk.
+As a result, in the Apple SDK, we track watchdog terminations during the app start based on heuristics, but getting the state of the app when a watchdog termination occurs is challenging. To provide the same state as we do for a typical crash, we would periodically need to store the app state to disk. This would cause a lot of I/O, which in turn could negatively affect the application's performance. To avoid this, watchdog termination events have less context than typical unhandled errors.
```

Comment:
> To provide the same state as we do for a typical crash, we would periodically need to store the app state to disk. This would cause a lot of I/O, which in turn could negatively affect the application's performance.

We should adapt this part as we have now breadcrumbs for them.